src/publish/bumpAndPush.ts

```diff
@@ -24,10 +24,15 @@ export async function bumpAndPush(bumpInfo: BumpInfo, publishBranch: string, opt
     // pull in latest from origin branch
     console.log('Fetching from remote');
     if (options.fetch !== false) {
-      if (options.depth) {
```

Comment:
Thanks for the contrib! Can you please run prettier on the code?

Reply:
done